unittests/gateway_service_test.yaml

```diff
@@ -26,7 +26,7 @@ tests:
       - equal:
           path: spec.type
           value: ClusterIP
-      - notExists:
+      - isNull:
```

Comment:
This seems unrelated to the PR. Is there any additional context for unit test update?